cmake/defaults/Packages.cmake

```diff
@@ -110,6 +110,9 @@ if(HoudiniUSD_FOUND)
 else()
     # We are using python to generate source files
     find_package(PythonInterp 3.7)
+
+    # Required, otherwise couldn't find it
+    set(OPENEXR_LOCATION ${USD_INCLUDE_DIR}/../)
```

Comment:
Could you pass this variable into cmake  instead of hardcoding it ? We setup this variable in Maya via passing it to cmake script